src/DataValues/MonolingualTextValue.php

```diff
@@ -68,16 +68,6 @@ public static function getType() {
 		return 'monolingualtext';
 	}
 
-	/**
-	 * @see DataValue::getSortKey
-	 *
-	 * @return string
-	 */
-	public function getSortKey() {
-		// TODO: we might want to re-think this key. Perhaps the language should simply be omitted.
-		return $this->languageCode . $this->text;
-	}
```

Comment:
Don't think you can do this without dropping compat with data-values/data-values, was was not done in this commit.

Reply:
Tests agree https://travis-ci.org/github/DataValues/Common/jobs/730260488

Reply:
Would just leave these here, doubt they will bother anyone. Not even sure these classes are used anywhere in the first place.

Reply:
I’m pretty sure `MonolingualTextValue` is used in Wikibase.